src/prommis/leaching/leach_reactions.py

```diff
@@ -10,10 +10,14 @@
 # All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
 # for full copyright and license information.
 #################################################################################
-"""
+r"""
```

Comment:
```suggestion
"""
```
Is this a typo?

Reply:
No - that is marking the following as a raw string, and is needed to avoid Python treating it as a regular expression.